extensions/dicom-html/src/DicomHtmlViewport.js

```diff
@@ -225,15 +233,34 @@ class DicomHtmlViewport extends Component {
     );
 
     this.setState({
-      content
+      content,
     });
   }
 
+  setViewportActiveHandler = () => {
+    const {
+      setViewportActive,
+      viewportIndex,
+      activeViewportIndex,
+    } = this.props;
+
+    if (viewportIndex !== activeViewportIndex) {
+      setViewportActive();
```

Comment:
If you're checking if the viewportIndexes are equal, is a throttle still required? This would short-circuit things pretty quickly, and may actually be more performant?

Reply:
I was afraid to let all scroll events happen and so I used the throttle only on `onScroll`, but in fact, with this condition, we get a quick escape if the method doesn't have to be executed. Maybe I can remove this? @dannyrb